client/components/lists/listBody.js

```diff
@@ -186,7 +186,7 @@ BlazeComponent.extendComponent({
           callback($.map(currentBoard.activeMembers(), (member) => {
             const user = Users.findOne(member.userId);
             return user.username.indexOf(term) === 0 ? user : null;
-          }));
+          }).filter(user => !!user));
```

Comment:
This should not be needed here, because jQuery map, will already filter out `null` values.